[14.0][MIG] utm

RaulOForgeFlow opened the pull request:
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

RaulOForgeFlow committed:
[14.0][MIG] utm

MiquelRForgeFlow commented on openupgrade_scripts/scripts/utm/14.0.1.0/upgrade_analysis.txt:
```diff
@@ -12,4 +12,4 @@ NEW utm.source: utm.utm_source_linkedin
 DEL utm.stage: utm.campaign_stage_1
 DEL utm.stage: utm.campaign_stage_2
-DEL utm.stage: utm.campaign_stage_3
+DEL utm.stage: utm.campaign_stage_3
```
I don't see what you changed here. Can you try to revert it?

MiquelRForgeFlow commented on openupgrade_scripts/scripts/utm/14.0.1.0/upgrade_analysis_work.txt:
```diff
@@ -0,0 +1,23 @@
+# NOTHING TO DO: noupdate = "0" records. Sample campaigns are now in utm_demo.xml
+
+
+
```
too much blank lines

flotho reviewed (approved):
AFAIK, looks good to me as there's mostly nothing to DO ;-)

pedrobaeza committed:
[IMP] utm: Delete some obsolete records

They were demo data not put as demo data, so we try to remove them if
possible.

pedrobaeza commented:
I have added a script for removing (if possible) that demo data not put as demo data in the previous version.
